src/common/Selects/PledgeFrequencySelect.tsx

```diff
@@ -0,0 +1,26 @@
+import { MenuItem, MenuItemProps, Select, SelectProps } from '@mui/material';
+import { PledgeFrequencyEnum } from 'src/graphql/types.generated';
+
+type PledgeFrequencySelectProps = Partial<SelectProps> & {
+  MenuItemProps?: Partial<MenuItemProps> & { [testid: string]: string };
```

Comment:
The component sets `data-testid="pledgeFrequencyOptions"`, but I don't see any tests using that test id, so it should be safe to remove. That will let us remove this `MenuItemProps` prop.